content/docs/services/cdn-route.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: CDN service
 name: "cdn-route"
-description: "Custom routes with HTTPS certificates using Amazon CloudFront"
+description: "Custom domains, CDN caching, and TLS certificates using Amazon CloudFront"
```

Comment:
This should be `Custom domains, CDN caching using Amazon CloudFront, and TLS certificates` since the domains and TLS certificates aren't using CloudFront